server.js

```diff
@@ -130,7 +130,7 @@ WITH day_0_base AS (
   SELECT
     *
   FROM
-    \`moz-fx-data-shared-prod.telemetry_derived.smoot_usage_nondesktop_compressed_v2\`
+    \`moz-fx-data-shared-prod.telemetry_derived.smoot_usage_nondesktop_compressed_v3\`
```

Comment:
The v3 tables are temporary. I'm staging these changes there for the purposes of being able to QA this change. I will remove this piece and recreate the v2 tables before merging.